rpc/json.go

```diff
@@ -136,24 +136,24 @@ func errorMessage(err error) *jsonrpcMessage {
 	return msg
 }
 
-type jsonError struct {
+type JsonError struct {
```

Comment:
Not entirely sure, but I think we could add a Go type alias (not a new type), to expose it, without changing the code of every receiver method here. That reduces the diff and improves maintainability.

Like so:
```go
type JsonError = jsonError
```
The `=` makes it the same underlying type, not a new type.

Reply:
neat! Didn't realize you can do this. Let me try